src/components/AddStallModal.tsx

```
@@ -273,46 +310,100 @@ export default function AddStallModal({ isOpen, onClose, onSubmit, initialData }
+        });
+
+        // Clear form data
+        setFormData({
+          name: '',
+          cuisine: '',
+          username: '',
+          password: '',
+          country: 'GERMANY',
+          profilePhoto: null,
+          timings: DAYS.reduce((acc, day) => ({
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)